lightning/src/ln/functional_tests.rs

```diff
@@ -6573,7 +6576,7 @@ fn test_update_add_htlc_bolt2_receiver_check_amount_received_more_than_min() {
 	nodes[1].node.handle_update_add_htlc(&nodes[0].node.get_our_node_id(), &updates.update_add_htlcs[0]);
 	assert!(nodes[1].node.list_channels().is_empty());
 	let err_msg = check_closed_broadcast!(nodes[1], true).unwrap();
-	assert_eq!(err_msg.data, "Remote side tried to send less than our minimum HTLC value");
+	assert!(err_msg.data.contains("Remote side tried to send less than our minimum HTLC value"));
```

Comment:
What criteria are you using when deciding between using `contains` (here) and using a regex `is_match` (e.g., lines 7440 and 7452)?

Reply:
I've first started without Regex, and later switched to Regex because It can be made more strict.
I probably should not use `contains` and use regex everywhere.